tests/ecmp/test_ecmp_sai_value.py

```diff
@@ -45,6 +45,22 @@
 offset_cmd = 'bcmcmd  "dump RTAG7_PORT_BASED_HASH 0 392 OFFSET_ECMP"'
 
 
+@pytest.fixture(autouse=True, scope='module')
+def enable_container_autorestart(duthosts, selected_rand_one_per_hwsku_hostname):
+    # Enable autorestart for all features before the test begins
+    for hostname in selected_rand_one_per_hwsku_hostname:
+        duthost = duthosts[hostname]
+        feature_list, _ = duthost.get_feature_status()
+        for feature, status in list(feature_list.items()):
+            if status == 'enabled':
+                duthost.shell("sudo config feature autorestart {} enabled".format(feature))
+    yield
+    # Config reload should set the auto restart back to state before test started
+    for hostname in selected_rand_one_per_hwsku_hostname:
+        duthost = duthosts[hostname]
+        config_reload(duthost, config_source='config_db', safe_reload=True)
```

Comment:
@vivekverma-arista one question: why choose reconfig_reload instead of disabling feature autorestart? You know, config reload is quite time consuming, especially when it's Dualtor, have to do config reload twice.

Reply:
Yes makes sense, I have fixed this.